selftest/tcpconnect/tcpclient

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sleep 5 ; echo "connected" | nc -q 2 127.0.0.1 12345 > /dev/null
```

Comment:
On fedora the `nc` that's packaged does not have a `-q` flag. The only similar options I see:

`--send-only            Only send data, ignoring received; quit on EOF`
`w, --wait <time>          Connect timeout`

Reply:
Hum, I'll change that usage then... did not expect that.